doc/nrf/releases_and_maturity/migration/migration_sysbuild.rst

```diff
@@ -0,0 +1,662 @@
+.. _child_parent_to_sysbuild_migration:
+
+Migrating from child/parent image to sysbuild
+*********************************************
+
+:ref:`sysbuild` is a build system used in zephyr used to configure, build, and flash multiple images as part of a single project.
+it replaces :ref:`child/parent images <ug_multi_image>` in |NCS|.
+As child/parent image has been deprecated,  you must update your projects to support being built using sysbuild.
+
+The following are the differences in how project configuration is performed in sysbuild compared to child/parent image configuration:
+
+* Sysbuild controls which images are added to a build, instead of the main application.
+
+* Sysbuild specifies the project configuration for all images, which was previously done in the main application or child applications.
+
+* Sysbuild controls the packaging of firmware components, instead of the main application.
+
+* Sysbuild manages some software functionality of image builds, such as the configuration mode of an nRF70-series radio.
+
+* Sysbuild runs partition manager code (if enabled) and distributes the output information to images.
```

Comment:
```suggestion
* Sysbuild controls which images are added to a build, rather than the main application.
* Sysbuild specifies the project configuration for all images.
  This was previously handled by the main application or child applications.
* Sysbuild controls the packaging of firmware components, rather than the main application.
* Sysbuild manages some software functionality of image builds, such as the configuration mode of an nRF70-series radio.
* Sysbuild executes partition manager code (if enabled) and distributes the output information to images.
```

Reply:
I had this before from your previous edits and it crushed all the text onto a single line without making it a list